tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
Unfortunately, tailwind is not part of this course. You have to practice your CSS skills and the only library that's been introduced is styled components. I'm sorry, I understand this means quite a lot of refactoring. If you have any questions, please come find me at the exercise class on Friday.